### ISSUE TITLE
update view wanneer er nieuwe resultaten binnenkomen

### ISSUE THREAD

**Issue by stijnbrysbaert** [stijnbrysbaert/thesis_demo#5]
Bij sommige queries (waarschijnlijk degene die wat langer duren) worden de resultaten niet allemaal getoond. Wanneer dan nog 1x op de `search` wordt gedrukt verschijnen de rest van de resultaten. Omdat de view dan handmatig geupdatet wordt?

**Comment by maartenbrysbaert** [comment]
Je moet altijd 2x duwen Probleempje waar ik al redelijk veel tijd aan gespendeerd heb, dus laat het voorlopig even zo..

**Comment by stijnbrysbaert** [comment]
Ja, maar het gebeurt soms dat je nog 1x moet drukken om de view nog eens te updaten zodat je alle resultaten kan zien omdat ze pas later binnenkomen dan de eerste resultaten.
